### DISCUSSION TITLE
Problem expanding fields with view rules

### DISCUSSION THREAD

**Discussion by mortegro** [pocketbase/pocketbase#4887] [Q&A]
The expansion of the manager in the room stops working, as soon as a view rule in the users collection is given. 
Viewing the user on the http endpoint works perfectly. 
Has anyone a clue, how this can be fixed? Thanks in advance...

```
pb.collection('rooms').getOne(id, {
         expand: 'manager'
}) 
```

![Snapshot_2024-05-09_12-54-26](https://github.com/pocketbase/pocketbase/assets/495066/242dd311-1028-4bbb-9a4b-ccec55128602)

![Snapshot_2024-05-09_12-52-48](https://github.com/pocketbase/pocketbase/assets/495066/56f1c07c-e4ee-432e-9894-0487c380ff37)

**Comment by ganigeorgiev** [comment]
It is working correctly. `expand` requires the request to satisfies the View API rule of the expanded collection (see the note in https://pocketbase.io/docs/working-with-relations/#expanding-relations). In your case this mean that you must be authenticated.
